tox.ini

```diff
@@ -23,27 +25,16 @@ deps =
     pinned: six==1.9.0
     pinned: werkzeug==0.9.6
 
-    latest: eventlet  # need something
+whitelist_externals = make
 
 commands =
-    pip install --editable .[dev]
-    flake8 nameko test
-    pylint --rcfile=pylintrc nameko -E
-    coverage run --concurrency=eventlet --source nameko -m \
-        pytest test {posargs:--timeout=30}
-    coverage report --show-missing --fail-under=100
+    codebase: pip install --editable .[dev]
```

Comment:
could we not just move `commands` out of the default section into a `[testenv:lib]`?